### ISSUE TITLE
Question: device change orientation

### ISSUE THREAD

**Issue by maoamid** [eclipsesource/tabris-js#1167]
Case: Specific layout for Landscape / Portrait  for multiple pages
How to prevent device.on change orientation to turn off the listener after the page is changed? Specific layouts for specific pages cannot be applied on other context. How I can determine the context where I should turn off the orientation listener? Maybe the page isn't changed but only the parent()...? Or I can turn on the listener only on specific context. How can I do that?

**Comment by mpost** [comment]
It is hard to understand the question could you rephrase it a bit? What do you try to achieve?

**Comment by maoamid** [comment]
@mpost Sorry, Let me rephrase: 
When I call `tabris.device.on('change:orientation', function(target, value)` on a page, after a leave it, the call remain. So, is impossible to change a specific page layout , because after I go to another one, if The device orientation is changed, the app try to appply the layout from the other page.

**Comment by mpost** [comment]
You can disable to listen with the `tabris.device.off(..)` call.

**Comment by maoamid** [comment]
@mpost This is the solution, but how to check the context when to disable / enable the call. I need some ideas in this direction, I don't know how to manage globally this calls in almost every page I have

**Comment by mpost** [comment]
Currently the `appear`/`disappear` events are your only option.

**Comment by tbuschto** [comment]
I think this can be considered answered.
